src/Components/DrillListPage.js

```diff
@@ -33,13 +37,14 @@ export const DrillListPage = props => {
           keyExtractor={item => item.id.toString()}
           renderItem={({ item }) => (
             <TouchableOpacity style={styles.drill} onPress={() => navigation.navigate('DrillPage', { drill: item })}>
-              <Image style={styles.image} source={{ uri: item.image }} />
+              <ImageBackground source={{ uri: item.image }} style={styles.image} imageStyle={styles.imageOpacity}>
+                <Text style={{ ...styles.imageText, ...styles.imageTextMain }}>{item[imageMainData]}+</Text>
+                <Text style={styles.imageText}>{imageMainDataLegend}</Text>
+              </ImageBackground>
               <View style={styles.contentContainer}>
                 <Text style={styles.source}>{item.source}</Text>
                 <Text style={styles.title}>{item.title}</Text>
-                <Text style={styles.numberOfPlayers}>
-                  Duration: {item.durationInMinutes} min - players: {item.nbPlayers}
```

Comment:
So we don't display drill duration anymore when it is a technical drill ?

Reply:
Nope, because yesterday we decided drill duration is not so important when talking about a technical drill (compared with fitness where duration is one of the most important piece of info)

But it's going to be displayed on the drill page.
Also, probably we should display it on the training page with a small drill list, since timing is important there

Reply:
Makes sense :)